stratum/hal/lib/barefoot/utils.h

```diff
@@ -47,6 +47,11 @@ ::util::StatusOr<int32> ConvertPriorityFromBfrtToP4rt(uint64 priority);
 // byte string.
 int NumBitsToNumBytes(int num_bits);
 
+// Converts unsigned integer to a byte array that fits certain bit width.
+// Returns error if the value is bigger than the size of bit width can store.
+::util::StatusOr<std::string> Uint32ToBytes(uint32 value, size_t bitwidth);
+::util::StatusOr<uint32> BytesToUint32(std::string value);
```

Comment:
Could `ByteStreamToUint` and `Uint32ToByteStream` cover these two functions? Main difference is the error handling.

Reply:
Didn't know we have this utility, will use that

Reply:
You can roll over your new test cases if you want.